workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/types.d.ts

```diff
@@ -0,0 +1,114 @@
+type Channel = {
+	currencyCode: string;
+	externalReferenceCode: string;
+	id: number;
+	name: string;
+	siteGroupId: number;
+	type: string;
+};
+
+type SKU = {
```

Comment:
@danvb112 I think you just need to declare the type with the properties you use in the code. Same applies for the types below.

Reply:
@naraandrade, I removed the unnecessary type definitions, just left the probably most important ones for the future